core/variant/callable.cpp

```diff
@@ -31,6 +31,8 @@
 #include "callable.h"
 
 #include "callable_bind.h"
+#include "variant_callable.h"
+
```

Comment:
Unless the order of includes matters here, I would make those two "absolute" (`core/variant/...`) and thus move them at the end after the `core/object` includes.